src/UICrafter.Client/Components/AppViewDesigner.razor

```diff
@@ -0,0 +1,87 @@
+@using static UICrafter.Client.Pages.UICrafterPage
+
+<MudContainer MaxWidth="MaxWidth.Small">
+    <MudDropContainer T="UIComponent" @ref="DropContainer" Items="UIComponents" ItemsSelector="@((item,dropzone) => item.DropZoneID == dropzone)" ItemDropped="ComponentUpdated" Class="d-flex flex-wrap flex-grow-1">
+        <ChildContent>
+            <MudDropZone T="UIComponent" Identifier="Drop Zone 1" AllowReorder="true" Class="rounded mud-background-gray pa-2 flex-grow-1" Style="height: 45rem; overflow: auto;">
```

Comment:
Overflow also exsists as css class if you are so inclined:
https://mudblazor.com/utilities/overflow#class-reference

Reply:
Sure :))